typescript/test/unit/api.ts

```diff
@@ -132,4 +140,16 @@ describe('schema/api.ts', () => {
       '../../protos/google/cloud/example/v1/test.proto',
     ]);
   });
+
+  it('should throw error when the service name is not found', () => {
+    assert.throws(() => {
```

Comment:
Can you please move just the `new API()` line into `assert.throws`? This unit test checks that that exact constructor throws an exception, if anything else in this block throws for whatever reason, it should be a test failure.

```ts
it(....., { 
  stuff;
  stuff;
  stuff;
  assert.throws(() => { new API(...); });
});
```

Reply:
agree